.claude/hooks/python_check/git_utils.py

```
@@ -16,31 +13,30 @@
 
 
 def run_git_command(
-    args: List[str], 
-    project_root: Path, 
-    capture_output: bool = True
+    args: list[str], project_root: Path, capture_output: bool = True
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)